tests/apps/pluggable_redis-statestore/go.mod

```diff
@@ -1,7 +1,9 @@
-module github.com/dapr/dapr/tests/apps/pluggable_redis
+module github.com/dapr/dapr/tests/apps/pluggable_redis-statestore
 
 go 1.19
 
+replace github.com/dapr/dapr => ../../../
```

Comment:
Remove replace.

Reply:
As dapr-sandbox uses a specific version of dapr protos, this replace will be necessary to make sure that the proposed change doesn't break the grpc stubs for the pluggable components.

If no replace is added means that the sdk will point to a dapr proto version that is always in conform with the grpc stubs.